Gifski/VideoValidator.swift

```diff
@@ -184,6 +184,20 @@ struct VideoValidator {
 			return .failure
 		}
 
-		return .success(newAsset, newVideoMetadata)
+		// Trim asset
+		do {
+			let trimmedAsset = try newAsset.trimBlankFrames()
+
+			return .success(trimmedAsset, newVideoMetadata)
+		} catch {
```

Comment:
You should not ignore the error. It should be included in the alert.

Reply:
Fixed in 50532fc.